source/ui/BioXAS/BioXASZebraLogicBlockInputInvertedView.cpp

```diff
@@ -0,0 +1,77 @@
+#include "BioXASZebraLogicBlockInputInvertedView.h"
+
+BioXASZebraLogicBlockInputInvertedView::BioXASZebraLogicBlockInputInvertedView(BioXASZebraLogicBlockInput *control, QWidget *parent) :
```

Comment:
This class is so alike with `BioXASZebraLogicBlockInputEnabledView`. I think we should generalize it.